mavis/main.py

```diff
@@ -620,20 +617,26 @@ def usage(err=None, detail=False):
     except AttributeError:
         pass
     # decide which main function to execute
-    if pstep == SUBCOMMAND.CLUSTER:
+    if args.command == SUBCOMMAND.CLUSTER:
         cluster_main(**args, start_time=start_time)
-    elif pstep == SUBCOMMAND.VALIDATE:
+    elif args.command == SUBCOMMAND.VALIDATE:
         validate_main(**args, start_time=start_time)
-    elif pstep == SUBCOMMAND.ANNOTATE:
+    elif args.command == SUBCOMMAND.ANNOTATE:
         annotate_main(**args, start_time=start_time)
-    elif pstep == SUBCOMMAND.PAIR:
+    elif args.command == SUBCOMMAND.PAIR:
         pairing_main(**args, start_time=start_time)
-    elif pstep == SUBCOMMAND.SUMMARY:
+    elif args.command == SUBCOMMAND.SUMMARY:
         summary_main(**args, start_time=start_time)
-    elif pstep == SUBCOMMAND.CONVERT:
+    elif args.command == SUBCOMMAND.CONVERT:
+        del args.command
```

Comment:
why do we need to del the args.command?

Reply:
so that I can pass args as an expansion and not have it thrown an unrecognized argument error